### ISSUE TITLE
Class Marketplace não faz parte dos releases

### ISSUE THREAD

**Issue by wesleysaraujo** [iugu/iugu-php#27]
Notei aqui que algumas classes que existe na branch master não estão entrando no Release, eu estava usando "^1.0" no meu release e não estava conseguindo usar a classe Marketplace, alterei para 1.0.6, ainda assim não estava encontrando a classe, após quebrar a cabeça aqui percebi que essa e mais outras classes não estão entrando nos releases (tags), precisei atualizar chamar "iugu/iugu" : "dev-master" para consegui usar a classe. Tem uma justificativa pra isso ou ainda tem um previsão dessas classes entrarem no próximo release.

**Comment by mchumer** [comment]
Olá Wesley,
estou na mesma situação que você. Implantei a versão do branch master do IUGU-PHP e estou usando.. 

Falei com o pessoal do IUGU e disseram que está no radar deles essa atualização. O PHP nem consta na nova documentação do API. Estão atrasados.

Fico na torcida para que andem rápido pq, realmente, usar esse pacote de forma improvisada é complicado.

**Comment by wesleysaraujo** [comment]
Sim, achei esse package bem fraquinho e defasado, eles não usam namespaces nem PSR-4, parece que tá bem abandonado.

**Comment by davimatos** [comment]
Mesma situação. =\

**Comment by mchumer** [comment]
Então.. o pessoal da IUGU disse que está no roteiro deles a atualização disso.
Por enquanto fiz o seguinte e funcionou.. já estou em produção.

Foi necessário instalar, via composer, a versão 1.06 e substituir os arquivos do pacote manualmente, além de liberar no gitignore para que leve para o servidor no commit / deploy.
nem sei se ficou correto, mas funciocionou, o que é bom indício, o meu gitignore. Acho que coloquei coisas a mais.

/vendor/*
!vendor/iugu
/iugu/*
!/iugu/iugu

Fato é que o está ignorando toda a minha pasta vendor com excessão da pasta do iugu, que está indo para o deploy.

Estou usando laravel - bitbucket - forge - aws

**Comment by mchumer** [comment]
Pessoal, vi que minha resposta foi mais de uma vez.. deixei só uma versão final, com mais detalhes.

abçs
